zoo/policies/open-agent/setup.py

```diff
@@ -1,11 +1,36 @@
 from setuptools import setup
+from pathlib import Path
 
-from open_agent.policy import VERSION
+from open_agent.version import VERSION
+
+try:
+    from open_agent.policy import compile_solver
+
+    compiled_extensions = compile_solver("./build")
+except ImportError:
+    compiled_extensions = []
```

Comment:
If this is important it might be useful to add a warning here.

Reply:
Done!